src/jaxsim/simulation/integrators.py

```diff
@@ -1,4 +1,5 @@
-from typing import Any, Callable, Dict, Tuple, Union
+import enum
+from typing import Any, Callable, Dict, Tuple
```

Comment:
Can we take advantage of this refactoring to replace `Dict` and `Tuple` with `dict` and `tuple`?

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/82/commits/29a6ba3e9eb3aa0f7597897aaa5eb843a1bf745e